query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/query/LdapRecommendedSettingsField.java

```diff
@@ -42,6 +42,14 @@
 
     public LdapRecommendedSettingsField() {
         super(DEFAULT_FIELD_NAME, FIELD_TYPE_NAME, DESCRIPTION);
+        userDns = new ListFieldImpl<>("userDns", LdapDistinguishedName.class);
+        groupDns = new ListFieldImpl<>("groupsDns", LdapDistinguishedName.class);
+        userNameAttributes = new ListFieldImpl<>("userNameAttributes", StringField.class);
+        groupObjectClasses = new ListFieldImpl<>("groupObjectClasses", StringField.class);
+        groupAttributesHoldingMember = new ListFieldImpl<>("groupAttributesHoldingMember", StringField.class);
```

Comment:
We use some of these string literals in `LdapSettingsField` too. Do these need to be in the service props classes?